src/SwedbankPay.Sdk/PaymentInstruments/Card/CardPaymentRequestObject.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+using SwedbankPay.Sdk.PaymentOrders;
+
+namespace SwedbankPay.Sdk.Payments.CardPayments
+{
+    public class CardPaymentRequestObject
+    {
+        public CardPaymentRequestObject(Operation operation,
+                                                Intent intent,
+                                                CurrencyCode currency,
+                                                List<IPrice> prices,
+                                                string description,
+                                                string payerReference,
+                                                bool generatePaymentToken,
+                                                bool generateReccurenceToken,
+                                                string userAgent,
+                                                CultureInfo language,
+                                                IUrls urls,
+                                                PayeeInfo payeeInfo,
+                                                RiskIndicator riskIndicator = null,
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)